code/_onclick/hud/human.dm

```diff
@@ -302,6 +302,15 @@
 	pull_icon.screen_loc = ui_pull_resist
 	static_inventory += pull_icon
 
+	mymob.fov = new /obj/screen()//I won't lie I have no fucking clue how TG's UI works.
+	mymob.fov.icon = 'icons/mob/vision_cone.dmi'
+	mymob.fov.icon_state = "combat"
+	mymob.fov.name = " "
+	mymob.fov.screen_loc = "1,1"
+	mymob.fov.mouse_opacity = 0
+	mymob.fov.layer = 17

```

Comment:
make a subtype of /obj/screen with all these properties and put it in screen_object.dm